### ISSUE TITLE
Cannot set current sound route to Speaker in latest version of QuickBlox-webRTC 2.2

### ISSUE THREAD

**Issue by astanton** [QuickBlox/quickblox-ios-sdk#561]
I see that now currentSoundRoute is a get-only property, and that there is now a setter.  However the setter does not seem to be doing anything.

This is the code I have right now is

```
QBRTCSoundRouter.instance().initialize()
QBRTCSoundRouter.instance().setCurrentSoundRoute(QBRTCSoundRoute.Speaker)
```

Before I upgraded from 2.1.1 to 2.2, this is what I had:

```
QBRTCSoundRouter.instance().initialize()
QBRTCSoundRouter.instance().currentSoundRoute = QBRTCSoundRoute.Speaker
```

And that worked fine.

I also noticed in the sample video chat the speaker sound also isn't working for me correctly, it's coming out of the receiver.

When I do 

```
print("---current sound route is \(QBRTCSoundRouter.instance().currentSoundRoute)")
```

After setting it to the speaker, this is the result I see:

```
---current sound route is QBRTCSoundRoute
```

Is my only option to downgrade back to 2.1.1 until this is fixed, or am I doing something wrong?

**Comment by Pro100Andrey** [comment]
Hi @astanton, please use 2.1.1. We fix this bug in new version of SDK

**Comment by astanton** [comment]
Is there any ETA on the release of the new SDK?  I see the last update was way back in March.

**Comment by adil-appifytech** [comment]
When we are expecting webrtc 2.3? If I am the caller, I am able to change my router to Speaker even on 2.1.1. Kind of having similar issue mentioned in this ticket https://github.com/QuickBlox/quickblox-ios-sdk/issues/415

**Comment by Raikerian** [comment]
@adil-appifytech everything is working fine in 2.1.1, you can check our [Q-municate app](https://github.com/QuickBlox/q-municate-ios) which is using it and have a correct speaker sound transfer.

New version is expected near the end of December.

**Comment by pethanihiten** [comment]
I have install 2.1.1 framework but it not found solution and not solution in 2.2 framework

**Comment by Raikerian** [comment]
Have you checked our Q-municate app? Does it work there?

**Comment by Raikerian** [comment]
Hi again @pethanihiten @adil-appifytech @astanton 

We have just released our new WebRTC SDK 2.3 which is containing completely new audio module.
You can now configure category, category options and mode of AVAudioSession used in WebRTC, read more here: http://quickblox.com/developers/Sample-webrtc-ios#Audio_Session_.28Previously_Sound_Router.29

`QBRTCSoundRouter` is deprecated from now on and you should use `QBRTCAudioSession`.
